index.d.ts

```diff
@@ -0,0 +1,13 @@
+// Type definitions for jasmine-data-provider 2.2.0
+// Project: https://github.com/MortalFlesh/jasmine-data-provider
+// Definitions by: Oleg Hoefling <https://github.com/hoefling>
+
+declare interface DataDescProvider<T> {
+  [key: string]: T;
+}
+
+declare function using<T>(data: T[], specDefinitions: (datum: T) => void): void;
```

Comment:
I'd rather don't use acronyms - especially in the declarations and definitions - `specDefinitions`, `DataDescProvider`, ...

Concrete and full names would be more explicit and more readable .. so for example something like
```typescript
ProviderWithDescription<T> {
    [description: string]: T;
}
```
or
```typescript
NamedDataProvider<T> {
    [caseName: string]: T;
}
```
or whatever makes more sense..